portal/server/routes/service-options.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const { validateToken } = require('./middleware');
+const { getServiceOptions } = require('../controllers/service-options');
+
+const router = express.Router();
+
+router.get('/', validateToken, (req, res) => res.redirect('/service-options'));
```

Comment:
Do you need to delete the equivalent of this in server/routes/dashboard.js line 16?

Reply:
Yes, I think it is deleted, it's just the test cases which need updating